src/openms/source/ANALYSIS/MAPMATCHING/PoseClusteringShiftSuperimposer.cpp

```diff
@@ -322,7 +322,7 @@ namespace OpenMS
 
         } // k
       } // i
-    } while (0);   // end of hashing (the extra syntax helps with code folding in eclipse!)
+    } while (false);   // end of hashing (the extra syntax helps with code folding in eclipse!)
```

Comment:
@timosachsenberg maybe we should remove extra control elements that only help with cosmetics?

Reply:
agree... this was not caught by clang-tidy but I only applied a few fixes so far. I can check if such nonsense can be removed automatically.